src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.MaxConnectionsPerServer.cs

```diff
@@ -58,16 +58,20 @@ public void Set_ValidValues_Success(int validValue)
             }
         }
 
-        [Fact]
-        public async Task GetAsync_Max1_ConcurrentCallsStillSucceed()
+        [Theory]
+        [InlineData(1, 5, false)]
+        [InlineData(1, 5, true)]
+        [InlineData(2, 2, false)]
+        [InlineData(2, 2, true)]
+        [InlineData(3, 2, false)]
+        [InlineData(3, 2, true)]
```

Comment:
Nit: Seems like it would be good to include a case like (3, 5) -- that is, more than one connection allowed but we exceed the limit.